ci/environment-3.8.yaml

```diff
@@ -1,4 +1,4 @@
-name: dask-ml-3.8
+name: dask-ml-latest
```

Comment:
Is this name change intentional, and if so, why not for the 3.9 environment yaml?

Reply:
Nope. Thanks!